packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -641,7 +647,7 @@ describe.each([
       const createdRow = await config.createRow()
 
       const res = await config.api.row.bulkDelete(table._id!, {
-        rows: [createdRow, { _id: "2" }],
+        rows: [createdRow, { _id: "9999999" }],
```

Comment:
This was occasionally failing for me with the value `"2"`, unsure why, suspect race condition.